### DISCUSSION TITLE
Suddenly can't view asset History.

### DISCUSSION THREAD

**Discussion by ozbugsy** [snipe/snipe-it#14595] [Q&A]
First, we are using a hosted version of snipe-it.

A week or so ago, one of my team members reported not being able to view the history of one particular asset.  I checked that asset and encountered the same issue - clicking on the history tab appears to do nothing.  At that time, other assets were still displaying history, and I had other priorities, so I put it on the back burner temporarily.   

Fast forward to today, and this issue is now affecting all assets (other item types - users, licences, etc are working as expected).

The affected users are all members of a group with Admin tier Permissions, so I asked one of our Super Admins to check and they can view history with no issues, so this appears to be a permissions issue.

I had the Super Admin user double-check the permissions of the affected Group (no I don't have Super Admin permissions, and yes I am the person who is being asked to fix it) - the Group my team members are assigned to is set to Admin tier and has the Reports: view permission set (I saw this mentioned in another post).

Has something changed recently, or is there something else I should be looking at?

Thanks.

**Comment by snipe** [comment]
@ozbugsy can you send an email to support@snipeitapp.com with a link to this issue and your hosted subdomain? We’ll be able to dig into the logs a little more if we have that info.

**Reply by ozbugsy** [reply]
Might be mostly user error - the history does appear - it just shows up under the information displayed on the info tab (and incidentally stays there when clicking back on info).  I will send the email now as well in case there is an issue that needs to be looked at :)
